neutrona/README.md

```diff
@@ -0,0 +1,78 @@
+# Agent Check: Neutrona
+
+## Overview
+
+This check monitors [Neutrona][1] cloud connectivity services to:
+ - Azure (ExpressRoute)
+
+## Setup
+
+### Installation
+
+The Neutrona check is not included in the [Datadog Agent][2] package, so you will
+need to install it yourself.
+
+You can copy the check files manually or run the _install.sh_ script if you are running the [Datadog Agent][2] on Debian/Ubuntu. This will copy both the check script and sample configuration file and **restart** the [Datadog Agent][2] :exclamation:. 
+
+
+### Configuration
+
+1. Edit the `neutrona.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your neutrona performance data.
+   See the [sample neutrona.d/conf.yaml][2] for all available configuration options.
+
+2. [Restart the Agent][3]
+
+### Validation
+
+[Run the Agent's `status` subcommand][4] and look for `neutrona` under the Checks section.
+
+## Data Collected
+
+### Azure Metrics
```

Comment:
The metrics should be in a metadata.csv file. Here is an example:
https://github.com/DataDog/integrations-extras/blob/master/aerospike/metadata.csv

Reply:
Metrics prefix missing.  [Fixed](https://github.com/DataDog/integrations-extras/pull/254/commits/f08c02d0c61fe838b49b3349df99c155cbb4c06b).

Reply:
Now the metrics list in the README should be replaced with:
See [metadata.csv][#] for a list of metrics provided by this integration.

And a link to the CSV should be added to the link list:
[#] https://github.com/DataDog/integrations-core/blob/master/neutrona/metadata.csv